src/App.js

```diff
@@ -1,25 +1,52 @@
-import logo from './logo.svg';
 import './App.css';
+import { Row, Divider, Button } from 'antd';
+import React, { useState } from 'react'
+import foods from './foods.json';
+import FoodBox from './Components/FoodBox';
+import AddFoodForm from './Components/AddFoodForm';
+import SearchBar from './Components/SearchBar';
+
 
 function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+  const [itens, setItens] = useState(foods);
+  const [showFood, setShowFood] = useState(foods);
+
+const deleteFood = (name) => {
+  const filtItens = itens.filter((banana) => banana.name !== name);
+  setItens(filtItens);
+  setShowFood(filtItens);
+};
+
+const createFood = (banana) => {
+  const newFood = [banana, ...itens]
+  setItens(newFood);
+}
+
+const filteredItens = (searchQuery) => {
+  let filtItens = itens.filter((banana) => 
+   banana.name.toLowerCase().includes(searchQuery.toLowerCase())
   );
+  setItens(filtItens); 
+}
+
+return (
+  <div className='App'>
+  <SearchBar filteredItens={filteredItens}/>
+  <Divider>Food List</Divider>
+  { itens.map ((allFoods) => {  
```

Comment:
now, you need to map over showFood, `{ showFood.map ((allFoods) => { ` instead of `itens.map`